ui/app/src/Utils/Views.elm

```diff
@@ -131,6 +134,56 @@ validatedField htmlField labelText classes inputMsg blurMsg field =
                 ]
 
 
+validatedTextareaField : (List (Attribute msg) -> List (Html msg) -> Html msg) -> String -> String -> (String -> msg) -> msg -> ValidatedField -> Html msg
+validatedTextareaField htmlField labelText classes inputMsg blurMsg field =
+    let
+        lineCount =
+            lines field.value
+                |> length
```

Comment:
`List` is always imported in Elm, so you can write `List.length` here and remove `import List exposing (length)`